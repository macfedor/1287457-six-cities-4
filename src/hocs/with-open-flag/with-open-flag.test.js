import * as React from "react";
import renderer from "react-test-renderer";
import withOpenFlag from "./with-open-flag";

const MockComponent = () => <div />;
const MockComponentWrapped = withOpenFlag(MockComponent);

it(`Should HOC render correctly`, () => {
  const tree = renderer
    .create(<MockComponentWrapped />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
