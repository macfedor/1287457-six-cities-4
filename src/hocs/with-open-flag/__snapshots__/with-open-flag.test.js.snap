// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should HOC render correctly 1`] = `<div />`;
